### DISCUSSION TITLE
Can I turn off parameter hints in the editor?

### DISCUSSION THREAD

**Discussion by noconsulate** [supabase/supabase#1122] [Questions] [answered]
I'd like to turn off the parameter hints/suggestions. They're really out of control and I have to hit esc every other word to make the editor usable. Very annoying! If I can't do this then please consider it a suggestion. :)

**Comment by silentworks** [comment, marked as answer]
This is currently not possible. Currently a possible workaround is to write your sql in a separate code editor or in a different SQL editor tool where you have control over this. You can connect to the supabase DB directly using the information found under **settings/database**.
